packages/system-tests/assets/sfdx-simple/.forceignore

```diff
@@ -1,2 +1,5 @@
 force-app/main/default/lwc/.gitkeep
-force-app/main/default/triggers/.gitkeep
\ No newline at end of file
+force-app/main/default/triggers/.gitkeep
```

Comment:
Lets revert all these changes under `packages/system-tests/assets/...` since they shouldn't be checked in.